workos/resources/audit_logs_export.py

```diff
@@ -0,0 +1,28 @@
+from workos.resources.base import WorkOSBaseResource
+from workos.resources.event_action import WorkOSEventAction
+
+
+class WorkOSAuditLogsExport(WorkOSBaseResource):
```

Comment:
This is what we've been calling the entity elsewhere.
```suggestion
class WorkOSAuditLogExport(WorkOSBaseResource):
```